tests/framework/web/ErrorActionTest.php

```
@@ -113,13 +113,12 @@ public function testDefaultView(): void
 
     public function testLayout(): void
     {
+        $ds = preg_quote(DIRECTORY_SEPARATOR, '\\');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ds. Configured minimum length is 3. (undefined)

(ShortVariable)